src/main/java/com/es/jointexpensetracker/service/impl/ExpenseServiceHardcodeImpl.java

```diff
@@ -0,0 +1,46 @@
+package com.es.jointexpensetracker.service.impl;
+
+import com.es.jointexpensetracker.model.Expense;
+import com.es.jointexpensetracker.service.ExpenseService;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+public class ExpenseServiceHardcodeImpl implements ExpenseService {
```

Comment:
Bad class name. Add new words at the beginning of the extended class. Or just one word "Impl" in the end.
Good names can be: HardcodeExpenseService or ExpenseServiceImpl